source/evaluating.html.erb.md

```diff
@@ -112,7 +112,7 @@ Plan to run evaluations as early as possible, even at design phase. This will no
 {:.attach_permalink}
 ## Run evaluation sessions with real people
 
-Seek feedback from people with disabilities on any materials being evaluated. This will help identify subtle issues, but will also emphasize the impact of any barriers. Encourage members of the project team to observe sessions as it is helpful in building understanding of the importance of accessibility.
+Seek feedback from people with disabilities on any materials being evaluated. This will help identify subtle issues, but will also emphasize the impact of any barriers. However,  be aware that everyone has different abilities with technology and a problem for one user may just be due to their technology or skill level. Encourage members of the project team to observe sessions as it is helpful in building understanding of the importance of accessibility.
```

Comment:
Point noted. Although I might have a think about how to present this. Any reasonable practitioner would have a range of participants and would be looking for common problems experienced by more than one individual.